test/parallel/test-dotenv-edge-cases.js

```diff
@@ -200,4 +200,26 @@ describe('.env supports edge cases', () => {
     assert.strictEqual(child.code, 9);
     assert.match(child.stderr, /bad option: --env-file-ABCD/);
   });
+
+  it('should handle invalid syntax in .env file', async () => {
+    const invalidEnvFilePath = fixtures.path('dotenv/invalid-syntax.env');
+    const code = `
+      const { parseEnv } = require('node:util');
+
+      const input = fs.readFileSync(${JSON.stringify(invalidEnvFilePath)}, 'utf8');
+
+      assert.deepStrictEqual(util.parseEnv(input), {
```

Comment:
nit: Wouldn't it make more sense to put the content here rather in a fixture file?

Reply:
The rest of tests use an fixture so I just do it as same